features/multiplication.feature

```diff
@@ -0,0 +1,20 @@
+Feature: Multiplication
+  As a mathematician
+  I want my calculator to multiply numbers
+  So I can know their product
+
+  Scenario: Multiplying numbers to get a product
+    When I multiply two numbers
+    Then I get the product
```

Comment:
The acceptance criteria requires that the product is an integer. You've not tested that here.